README.md

```diff
@@ -28,61 +28,46 @@ OpenCensus libraries require Go 1.8 or later as it uses the convenience function
 
 ## Tags API
 
-### To create/retrieve a key
-A key is defined by its name. To use a key a user needs to know its name and type (currently only keys of type string are supported. Later keys of type int64 and bool will be supported). Calling CreateKeyString(...) multiple times with the same name returns the same key.
+Tags represent propagated key values. They can propagated context.Context in the same process or can be encoded
+to be transmitted on wire and decoded back to a TagSet at the destination.
 
-Create/retrieve key:
+### Getting a key by a name
+A key is defined by its name. To use a key a user needs to know its name and type. Currently, only keys of type string are supported.
+The library will support int64 and bool keys in the future.
```

Comment:
I would say. other types will be supported in the future. Up to you.

Reply:
Done